samples/feature_flag_with_azure_app_configuration_sample.py

```diff
@@ -0,0 +1,45 @@
+from microsoft.featuremanagement import FeatureManager
+from random_filter import RandomFilter
+from azure.appconfiguration.provider import load, SettingSelector
+import os
+from sample_utilities import get_authority, get_audience, get_credential, get_client_modifications
+import json
+import os
+import sys
+from time import sleep
+
+endpoint = os.environ.get("APPCONFIGURATION_ENDPOINT_STRING")
+authority = get_authority(endpoint)
+audience = get_audience(authority)
+credential = get_credential(authority)
+kwargs = get_client_modifications()
+
+# Connecting to Azure App Configuration using AAD
+config = load(
+    endpoint=endpoint, credential=credential, feature_flag_enabled=True, feature_flag_refresh_enabled=True, **kwargs
```

Comment:
Does the configuration provider not take connection string directly?

Reply:
It can, this is AAD. I can make it use a connection string the same way.